.github/workflows/main.yml

```diff
@@ -11,15 +11,3 @@ env:
 jobs:
   test:
     uses: SiaFoundation/workflows/.github/workflows/go-test.yml@master
-  analyze:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Jape Analyzer
-        uses: SiaFoundation/action-golang-analysis@HEAD
-        with:
-          analyzers: |
-            go.sia.tech/jape.Analyzer@master
-          directories: |
-            api
```

Comment:
Are you removing this to get around encoding `nil`?

Reply:
Yup. GET Endpoints should be able to return 204.